src/tree.js

```diff
@@ -1168,7 +1168,9 @@ let tree = (function(){
                 ret = ret.substring(0, ret.length-postfix.length).trim();           
             postfix =  'and\n';   
             if( 0 < ret.indexOf(postfix) && ret.indexOf(postfix) == ret.length-postfix.length )
-                ret = ret.substring(0, ret.length-postfix.length).trim();           
+                ret = ret.substring(0, ret.length-postfix.length).trim(); 
+            if( !ret.endsWith('/n') )
```

Comment:
Is this supposed to be checking for if it does NOT end with \n (not /n) then to add it?

```suggestion
            if( !ret.endsWith('\n') )
```

Reply:
That is my only question just want to confirm - otherwise all set. Thanks Vadim!